<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E2R_every_report</class>
 <widget class="QWidget" name="E2R_every_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="export_btn">
     <property name="text">
      <string>파일추출</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>556</width>
        <height>677</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="18" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="16" column="3">
        <widget class="QPushButton" name="daily_pad_ashing_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="mesure_1_path"/>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Daily PAD 막두께</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Daily PAD 에싱후 
PR두께</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QLineEdit" name="daily_gap_path"/>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>3매측정_PR_CD관리</string>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QLineEdit" name="Daily_Bridge_path"/>
       </item>
       <item row="15" column="2">
        <widget class="QLineEdit" name="Daily_pad_path"/>
       </item>
       <item row="16" column="2">
        <widget class="QLineEdit" name="daily_pad_ashing_path"/>
       </item>
       <item row="17" column="2">
        <widget class="QLineEdit" name="mesure_pr_cd_path"/>
       </item>
       <item row="13" column="3">
        <widget class="QPushButton" name="daily_gap_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QPushButton" name="Daily_Bridge_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="15" column="3">
        <widget class="QPushButton" name="Daily_pad_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="17" column="3">
        <widget class="QPushButton" name="mesure_pr_cd_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="mesure_1_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>1차 PR측정 관리대장</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>2차,3차 PR두께 측정</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="SIO2_Etch_rate_path"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>SIN_Etch_rate관리</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ICP_Etch_rate_관리</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="ICP_Etch_rate_path"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="ICP_Etch_rate_btn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="SIN_Etch_rate_path"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="SIN_Etch_rate_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>SiO2_Etch_rate_관리</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>export 파일 경로</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="export_path_edit"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="export_path_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>WLP SiO2 막두께</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="WPL_SiO2_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>ICP_Ashing_rate_관리</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="ICP_Ashing_rate_path"/>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="ICP_Ashing_rate_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="mesure_2_3_path"/>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="mesure_2_3_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>성막일일관리</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="sungmak_day_path"/>
       </item>
       <item row="11" column="3">
        <widget class="QPushButton" name="sungmak_day_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>설비별 균일도 check</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QLineEdit" name="uniformity_ckeck_path"/>
       </item>
       <item row="12" column="3">
        <widget class="QPushButton" name="uniformity_ckeck_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Daily GAP Trend</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Daily Bridge 두께</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="SIO2_Etch_rate_btn">
         <property name="text">
          <string>파일경로</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="WPL_SiO2_path"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="excel_mecro">
     <property name="text">
      <string>mecro</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
